src/styles/Utils/mixins.scss

```diff
@@ -0,0 +1,14 @@
+@mixin centererd-hands() {
+  position: absolute;
+  bottom: 40vmin;
+}
+
+@mixin speed-up() {
```

Comment:
It's a good practice to leave a comment describing what the mixin does or when it should be used. This helps with maintainability and understanding the codebase for both current and future developers.